index.d.ts

```diff
@@ -0,0 +1,40 @@
+declare module 'envvar' {
+  /**
+   * Retrieves a given environment variable as a string.
```

Comment:
Actually, I like _specified_ in place of _given_ as we're not given an environment variable; we're told the name of an environment variable so we can retrieve its value from `process.env`.